api/src/main/java/jakarta/data/repository/Sort.java

```diff
@@ -63,7 +63,7 @@ private Sort(String property, Direction direction) {
     /**
```

Comment:
Should we also replace `Direction direction` with `boolean isAscending` to align with Java records?
```
public record Sort(String property, boolean isAscending) {
    public boolean isDescending() {
        return !isAscending();
    }

    public static Sort asc(String property) {
        return new Sort(property, true);
    }

    public static Sort desc(String property) {
        return new Sort(property, false);
    }
}
```